test/CMakeLists.txt

```diff
@@ -315,6 +315,7 @@ set(MULTI_TEST_SRC
     backend/batch_norm.in.cpp
     backend/broadcast.in.cpp
     backend/builder_flatten.in.cpp
+    backend/builder_opset1.in.cpp
```

Comment:
Maybe renaming it `builder_reduce_ops_opset1.in.cpp`  would better fit to naming convention as file name in line above?

Reply:
good point